src/documentation/pages/components/tabs/examples/_icons-horizontal-lumen-centurylink.vue

```diff
@@ -0,0 +1,125 @@
+<template lang="pug">
+  <ComponentTabExample title="Horizontal tabs with icons" id="icons-horizontal-lumen-centurylink" padding="-p--0" :tabs="exampleTabs" :menuTabs="menuTabs" :menuId="menuId" @toggleMenuId="toggleMenuId">
+    .-p--3.-bg--grey-20(:slot="$data.menuTabs[0].id")
```

Comment:
i believe you could use additionalClasses here (and in other examples) for the bg color